web/tests/functional/report_viewer_api/__init__.py

```diff
@@ -58,7 +58,8 @@ def setup_package():
         'check_env': test_env,
         'workspace': TEST_WORKSPACE,
         'checkers': [],
-        'tag': tag
+        'tag': tag,
+        'analyzers': ['clangsa', 'clang-tidy']
```

Comment:
shouldn't we add these to all of the test files, to explicitly set what analyzers should run for that analysis.